test/cur.jl

```diff
@@ -34,7 +34,7 @@ module LMDB_CUR
     @test !isopen(env)
 
     # Block style
-    open(dbname) do env # open environment
+    open(LMDBPath(dbname)) do env # open environment
```

Comment:
It is quite ugly to wrap a string into an object just to pass it as an argument. Maybe, you should rename `open` function to `environment`. This will give clear description of what function does.

Reply:
Yeah... I too felt `open` was kind of too generic. A new exported function may be better instead.